src/main/java/org/sagebionetworks/web/client/widget/docker/modal/AddDockerCommitModalViewImpl.java

```diff
@@ -0,0 +1,107 @@
+package org.sagebionetworks.web.client.widget.docker.modal;
+
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+import org.gwtbootstrap3.client.ui.Button;
+import org.gwtbootstrap3.client.ui.Modal;
+import org.gwtbootstrap3.client.ui.TextBox;
+import org.gwtbootstrap3.client.ui.html.Div;
+
+public class AddDockerCommitModalViewImpl implements AddDockerCommitModalView {
+
+  public interface Binder
+    extends UiBinder<Widget, AddDockerCommitModalViewImpl> {}
+
+  @UiField
+  Button saveButton;
+
+  @UiField
+  Button cancelButton;
+
+  @UiField
+  Modal addDockerCommitModal;
+
+  @UiField
+  TextBox digestTextBox;
+
+  @UiField
+  TextBox tagTextBox;
+
+  @UiField
+  Div synAlertContainer;
+
+  private Widget widget;
+  private Presenter presenter;
+
+  @Inject
+  public AddDockerCommitModalViewImpl(Binder binder) {
+    widget = binder.createAndBindUi(this);
+    saveButton.addClickHandler(
+      new ClickHandler() {
```

Comment:
These could be converted to lambda expressions to make more concise (since ClickHandler only has the one method).

```
saveButton.addClickHandler(event -> presenter.onSave());
```